tests/MongoDB.EntityFrameworkCore.FunctionalTests/Serialization/MongoTypeSerializationTests.cs

```diff
@@ -0,0 +1,181 @@
+﻿/* Copyright 2023-present MongoDB Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using MongoDB.Bson;
+
+namespace MongoDB.EntityFrameworkCore.FunctionalTests.Serialization;
+
+public class MongoTypeSerializationTests : BaseSerializationTests
+{
+    public MongoTypeSerializationTests(TemporaryDatabaseFixture tempDatabase)
+        : base(tempDatabase)
+    {
+    }
+
+    [Theory]
+    [InlineData("652446393021fe289cf2c197")]
+    [InlineData("64a8583aa1ee84d292c009dd")]
```

Comment:
This test cases are seems to be equivalent for me.

Reply:
UUID formats